packages/keymaps/src/browser/keybindings-widget.tsx

```diff
@@ -211,7 +211,7 @@ export class KeybindingWidget extends ReactWidget implements StatefulWidget {
 
     protected formatAndMatchKeybinding(item: KeybindingItem, queryItems: string[]): boolean {
         if (item.keybinding) {
-            let match = false;
+            const searchTerms = [...queryItems].filter(Boolean);
```

Comment:
Since `filter` always creates a new array, there's no need to spread here.

```suggestion
            const searchTerms = queryItems.filter(Boolean);
```